### DISCUSSION TITLE
Pip plugin doesn't use current python version

### DISCUSSION THREAD

**Discussion by andreamoro** [ohmyzsh/ohmyzsh#11945] [Help]
Have been trying to read as many post as possible (not issues, which apparently haven't this covered), but couldn't find a real solution.

My Mac over time got multiple versions of python and pip installed, but only recently with brew.

When attempting to pip a module, the package looks for pip in the system default python directory, which as a result installs all the libraries in the "wrong" place (and later reference the wrong python in the virtualenv).

That said, as far as I can see, it seems the pip in the cellar to have the wrong path reference, but leave this here to raise the attention and perhaps work out on a solution?

**Comment by msafadieh** [comment]
Are you talking about Python 3 vs Python 2 differences? In other words, do you want to install modules to use with Python 3 but instead download the Python 2 ones?

You should be able to use `pip3` for that, no?

**Comment by andreamoro** [comment]
Well, that was the issue, there were even no pip3 installed. So ended up cleaning every installation and starting from scratch.
This solved my problem for sure, but I'm still considering the fact the script sound be more generic and try to get the latest version rather than hardcode the reference
